src/poliastro/core/perturbations.py

```diff
@@ -217,6 +217,32 @@ def third_body(t0, state, k, k_third, perturbation_body):
     k : float
         gravitational constant, (km^3/s^2)
     perturbation_body: a callable object returning the position of the pertubation body that causes the perturbation
+        .. py:function:: perturbation_body() -> position_of_perturbation_body
+        .. py:class:: Perturbation_body() -> position_of_perturbation_body
+
+    Example
+    -------
+    For clarifying the nature of **perturbation_body** argument in **third_body()** function, we can look into the following example:
+
+    >>> from poliastro.ephem import build_ephem_interpolant
+    >>> from poliastro.bodies import Moon
+    >>> from astropy.time import Time, TimeDelta
+    >>> epoch = Time(
+    ...    2454283.0, format="jd", scale="tdb"
+    ...     )  # setting the exact event date is important
+    >>> body_r = build_ephem_interpolant(
+    ...   Moon, 28 * u.day, (epoch.value * u.day, epoch.value * u.day + 60 * u.day), rtol=1e-2
+    ... )
+    >>> perturbation_body = body_r      # passing created callable function to perturbation_body
```

Comment:
This line is not needed? It's just renaming a variable

Reply:
Same as above ... making it relevant to the user based on the function call signature :)